openqabot/types/incident.py

```diff
@@ -144,3 +148,30 @@ def contains_package(self, requires: List[str]) -> bool:
                 if package.startswith(req) and package != "kernel-livepatch-tools":
                     return True
         return False
+
+    def has_failures(self, token) -> bool:
+        results = request.get(
+            QEM_DASHBOARD + "/api/jobs/incident/" + str(self.id), headers=token
+        ).json()
+
+        # This should be wrapped by a verbose flag so we return faster
+        for res in results:
+            if res["status"] == "passed":
+                continue
+            else:
```

Comment:
This seems redundant. How about just

```suggestion
            if res["status"] != "passed":
```

Reply:
See above, I don't put too much brainpower on thtis

Reply:
That does not resolve the discussion point though. I might be wrong then please prove me wrong, otherwise my suggestion holds

Reply:
@okurz rather than wasting time over style issues, lets focus on the important part of the problem:

can you guide me in the test/code paths? `the_demon_incident` in theory should not show up, any thoughts?